api/src/services/products/products.js

```diff
@@ -10,13 +11,14 @@ export const products = async () => {
 
     return {
       // product data
-      id: product.id,
+      id: price.id,
       name: product.name,
       description: product.description,
       images: product.images,
       // price data
       price: price.unit_amount,
-      priceId: price.id,
+      productId: product.id,
+      type: price.type,
```

Comment:
What was the motivation for the switch here? That is the `id` now being the `price.id`? I know we don't really use the `productId` for much yet but it seems a little weird that the `Product` type's `id` property isn't actually its id. Maybe I'm overthinking it?

Reply:
Yeah the motivation is that the `id` is now `price.id`. It is a bit weird but I think that's just the way things are now...